app/models/trip.rb

```diff
@@ -0,0 +1,4 @@
+class Trip < ApplicationRecord
+  belongs_to :driver
+  belongs_to :passenger
```

Comment:
There are no validations on Trip creation. Either add some validations here, or handle in your controller what happens if there's no date or cost passed into the form